site/docs/v1/tech/apis/system.adoc

```diff
@@ -415,21 +415,25 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [method]#GET# [uri]#/api/prometheus/metrics#
 --
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#basic-authentication[icon:shield[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
 === Authentication
-This endpoint can be authenticated with either API Keys or `basic auth` with API Keys authentication methods. Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about how to use the `basic auth` authentication method with API keys.
+As of version 1.27.0 this endpoint can be authenticated with an API key or `basic auth`. Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about how to use `basic auth`.
```

Comment:
Do we need this blurb? If we show both methods above - we should be good? I'm not too concerned about the version thing since we mostly fixed this right away.

Reply:
Sure thing, will remove.